CONTRIBUTING.md

```diff
@@ -83,12 +71,11 @@ We use the following terminology:
 
 ### Language Track Repositories
 
-We've given each language track an ID, which is a url-friendly version of the
-language name. For example, _C++_ has the ID `cpp`. This ID is used throughout
-the Exercism ecosystem.
+We have given each language track an ID, which is an url-friendly version of the language name.
```

Comment:
`an url-` vs `a URL-`?
The latter seems more correct to me. But it's not a big deal.